pandas/core/reshape/reshape.py

```diff
@@ -728,6 +728,11 @@ def get_dummies(data, prefix=None, prefix_sep='_', dummy_na=False,
 
         .. versionadded:: 0.18.0
 
+    dtype : dtype, default np.uint8
```

Comment:
Can we also accept arguments to `np.dtype` like the string `'i8'`, and handle those appropriately?

Reply:
he is using ``np.dtype()``?

Reply:
Yes, that should work already, I'll add it to the tests.